src/main/config/config.h

```diff
@@ -26,6 +26,7 @@
 #include "pg/pg.h"
 
 #define MAX_NAME_LENGTH 16u
+#define OSD_CUSTOM_MSG_COUNT 4
```

Comment:
@ledvinap , @haslinghuis , what about
```C
#ifndef OSD_CUSTOM_MSG_COUNT
#define OSD_CUSTOM_MSG_COUNT 4
#endif
```

Reply:
Would suggest moving to osd_elements.h

Reply:
But it is saved in config, so it may cause dependency problems

Reply:
MSP seems to be handling 8, not 4 messages.

Reply:
even 4 is probably more than enough  but it is easy to build version with (up to) 8. But if there is no memory pressure, we can default to maximum

Reply:
Thanks, I have moved the macro define  OSD_CUSTOM_MSG_COUNT to osd.h . And its enough to set maximum as 4, in my own project.